src/backend/marsha/core/signals/handlers.py

```diff
@@ -0,0 +1,42 @@
+"""Marsha's module containing signals receiver."""
+
+from ..defaults import STOPPED
+from ..utils.medialive_utils import medialive_client, mediapackage_client
+
+
+# pylint: disable=unused-argument
+def delete_aws_element_stack(sender, **kwargs):
```

Comment:
```suggestion
def delete_aws_element_stack(sender, video, **kwargs):
```

Reply:
according to the documentation I can not do that.